boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts

```diff
@@ -83,14 +83,6 @@
 		regulator-boot-on;
 	};
 
-	nrf21540fem_ctrl: nrf21540fem-ctrl {
-		compatible = "nordic,nrf21540-fem";
-		rx-en-gpios = <&gpio1 11 GPIO_ACTIVE_HIGH>;
-		mode-gpios  = <&gpio1 12 GPIO_ACTIVE_HIGH>;
-		pdn-gpios   = <&gpio1 10 GPIO_ACTIVE_HIGH>;
-		tx-en-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
-	};
-
```

Comment:
> > The only thing i find inconsistent is having same pins in NET core as @jciupis suggested.
> 
> @emob-nordic could you elaborate, please?

Sorry I introduced some chaos. I think FEM pins should remain here and in lines 182-189 as this is the default configuration for `Thingy:53`, i was refering to this:

> Do you think it'd make sense to add a relevant, matching entry in application's core .dts file?